wcfsetup/install/files/lib/system/user/notification/object/type/IMultiRecipientCommentResponseOwnerUserNotificationObjectType.class.php

```diff
@@ -5,24 +5,8 @@
 use wcf\data\comment\Comment;
 
 /**
- * Default interface for comment user notification object types with notifications
- * being sent to multiple recipients and has a `commentResponseOwner` event.
- *
- * This interface is only required, if you use the interface `IMultiRecipientCommentUserNotificationObjectType`.
- * If you use not this interface, the `commentResponseOwner` event is fired by default.
- *
- * @author  Joshua Ruesweg
- * @copyright   2001-2019 WoltLab GmbH
- * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
- * @since   5.2
+ * @deprecated
```

Comment:
Please include the version number and a brief summary of what to do if one implements this interface.